lib/perl/Genome/Site/TGI/Extension/Sys.pm

```diff
@@ -92,15 +88,17 @@ sub bsub_and_wait_for_completion {
             %this_cmd_bsub_args = ( cmd => $cmd );
         }
 
-        my $job_id = $class->bsub(post_exec_cmd => qq(echo $seq | netcat $hostname $port),
+        my $job_id = $class->bsub(
                                   %bsub_args,
                                   %this_cmd_bsub_args,
                                 );
         $on_submit_cb->($seq, $job_id) if $on_submit_cb;
         $seq_to_job_id{$seq} = $job_id;
     }
 
-    $class->_bsub_and_wait_for_completion__wait_on_jobs($waiting_socket, $on_complete_cb, %seq_to_job_id);
+    $class->_bsub_and_wait_for_completion__wait_on_jobs_with_lsf_dependency( \%seq_to_job_id,
+        { queue => $bsub_args{queue} || 'short' },
```

Comment:
This should not hardcode an LSF queue.  We have:
```perl
Genome::Config::get('lsf_queue_short')
```

I would advocate that this always use that queue instead of whatever queue was used for the other jobs.  (For instance, this waiting job needn't go to the `hpss` queue.)